pcsx2/Recording/VirtualPad/VirtualPad.cpp

```diff
@@ -116,8 +116,10 @@ VirtualPad::VirtualPad(wxWindow* parent, int controllerPort, AppConfig::InputRec
 	SetBackgroundColour(*wxWHITE);
 	SetBackgroundStyle(wxBG_STYLE_PAINT);
 	// This window does not allow for resizing for sake of simplicity: all images are scaled initially and stored, ready to be rendered
-	SetWindowStyle(wxDEFAULT_FRAME_STYLE & ~wxRESIZE_BORDER);
-	SetDoubleBuffered(true);
+	
+	//SetWindowStyle();
+	//Refresh();
+	SetDoubleBuffered(false);
```

Comment:
We'll have to see how my solution compares to disabling double buffering.  Double-buffering is actually enabled by default on some platforms as well (linux/mac) I can't tell you for sure _why_ this fixed it for you, but I don't think it's the only way to fix it.  Nice point to compare it to.

Reply:
Commenting this out does indeed seem to be the right way to go, even combined with my new buffered' DC instances.  I really hope this style of GUI programming is easier in Qt lol.